agent/lib/kontena/workers/stats_worker.rb

```diff
@@ -151,7 +156,7 @@ def send_statsd_metrics(name, event)
       if labels && labels[:'io.kontena.service.name']
         key_base = "services.#{name}"
       else
-        key_base = "#{node_name}.containers.#{name}"
+        key_base = "#{@node_name}.containers.#{name}"
```

Comment:
Oh, right... but it smells a bit to have `@node_name` instance variable.. why not store whole `@node`?